aspnetcore/fundamentals/servers/aspnet-core-module.md

```diff
@@ -0,0 +1,85 @@
+
+ANCM has a few other functions as well:
+
+* Sets environment variables.
+* Logs `stdout` output to file storage.
+* Forwards Windows authentication tokens.
+
+## How to use ANCM in ASP.NET Core apps
+
+This section provides an overview of the process for setting up an IIS server and ASP.NET Core application. For detailed instructions, see [Publishing to IIS](../../publishing/iis.md).
+
+### Install ANCM
+
+The ASP.NET Core Module has to be installed in IIS on your servers and in IIS Express on your development machines. For servers, ANCM is included in the [ASP.NET Core Server Hosting Bundle](http://go.microsoft.com/fwlink/?LinkId=827547). For development machines, Visual Studio automatically installs ANCM.
+
+### Install the IISIntegration NuGet package
+
+In your application, install [Microsoft.AspNetCore.Server.IISIntegration](https://www.nuget.org/packages/Microsoft.AspNetCore.Server.IISIntegration/). This is an interoperability pack that reads environment variables broadcast by ANCM to set up your app. The environment variables provide configuration information such as the port to listen on. 
+
+### Call UseIISIntegration
+
+In your application's `Main` method, call the `UseIISIntegration` extension method on [`WebHostBuilder`](http://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNetCore/Hosting/WebHostBuilder/index.html#Microsoft.AspNetCore.Hosting.WebHostBuilder.md). 
+
+[!code-csharp[](aspnet-core-module/sample/Program.cs?name=snippet_Main&highlight=12)]
+
+The `UseIISIntegration` method looks for environment variables that ANCM sets, and it does nothing if they aren't found. This behavior facilitates scenarios like [developing and testing on MacOS and deploying to a server that runs IIS](../../tutorials/your-first-mac-aspnet.md).  While running on the Mac, Kestrel acts as the web server, but when the app is deployed to the IIS environment, it automatically hooks up to ANCM and IIS.
+
+### Don't call UseUrls
+
+When you use ANCM, don't call the `UseUrls` method, because the `UseIISIntegration` method picks up the port to listen on from environment variables that ANCM sets. When you run the app without IIS, it listens on the default port number at http://localhost:5000.
+
+If you need to set the port number for when you run the app without IIS, call `UseURLs` **before** you call `IISIntegration`.  When you run without IIS, the port number that you provide to `UseUrls` will take effect because `IISIntegration` will do nothing. But when you run with IIS, the port number specified by ANCM will override whatever you passed to `UseUrls`.
```

Comment:
In 1.1 the ordering is no longer a problem because UseIISIntegration will now overwrite UseUrls regardless of the order they are called in.